Src/AutoNSubstituteUnitTest/SubstituteAttributeTest.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Linq;
+using Xunit;
+using Xunit.Extensions;
+
+namespace Ploeh.AutoFixture.AutoNSubstitute.UnitTest
+{
+    public class SubstituteAttributeTest

```

Comment:
You decide :smile: 

I don't think it adds any value, but I don't think it hurts either.